src/app/client/src/app/modules/public/module/player/components/public-collection-player/public-collection-player.component.ts

```diff
@@ -338,16 +338,7 @@ export class PublicCollectionPlayerComponent implements OnInit, OnDestroy, After
       sessionStorage.setItem('singleContentRedirect', 'singleContentRedirect');
       this.router.navigate(['/get/dial/', this.dialCode]);
     } else {
-      if (this.isOffline) {
-       const  previousUrl =  this.navigationHelperService.getPreviousUrl();
-       if (Boolean(_.includes(previousUrl.url, '/play/collection/'))) {
-        return this.router.navigate(['/']);
-       }
-       // tslint:disable-next-line: max-line-length
-       previousUrl.queryParams ? this.router.navigate([previousUrl.url], {queryParams: previousUrl.queryParams}) : this.router.navigate([previousUrl.url]);
-      } else {
-        this.navigationHelperService.navigateToResource('/explore');
-      }
+      this.navigationHelperService.navigateToPreviousUrl('/explore');
```

Comment:
You have removed complete offline check here. Is it not require now?

Reply:
I have moved offline code to [desktop-repo ](https://github.com/Sunbird-Ed/sunbird-desktop-app/tree/master/src/client)in the previous release. We need to remove all references of offline code in all the components.